data/remnant/remnant 2 cognizance.txt

```diff
@@ -177,8 +174,7 @@ mission "Remnant: Cognizance 4"
 		system "Aescolanus"
 		fleet
 			names "korath"
-			cargo 2
-			commodities "void sprite parts"
+			cargo 3
```

Comment:
Likewise, I'd prefer to ensure the intended behavior is happening, rather than replacing it with a non-intended generic behavior.